### DISCUSSION TITLE
Get SMPlayer on Linux to refer to Yt-dlp in a non default location?

### DISCUSSION THREAD

**Discussion by octimis** [smplayer-dev/smplayer#853] [Q&A]
I have yt-dlp installed in: /home/username/Applications/yt-dlp_linux
SMPlayer insists on installing when I do Help> Install/Update YouTube support: /home/username/bin/yt-dlp

I'd rather not have redundant apps if it can be avoided.

**Comment by dvd73** [comment]
Use [alias](https://www.rapidseedbox.com/blog/yt-dlp-complete-guide)

**Comment by bendover22** [comment]
@octimis, where is, ~/username/**Applications**/ folder?  In Mint, there's no ~/Applications (nor .applications).
There's ~/.local/share/applications/ ; that stores desktop files (app launchers).

What's wrong w/ using the dev's default locations?  (in most 'Nix, default is usually a safe bet).
Not sure that **SMTube** installed to (think it should say):  ~/.local/bin/yt-dlp, is a "duplicate."  
SMTube is separate.

In LM, yt-dlp base prog installs to: /usr/bin/yt-dlp.   
One **python file** is in ~/.local/bin/yt-dlp (as you show).
And its conf file in ~/.config/yt-dlp/config
